### PR TITLE
Add my items (mocked) and Improvements for mobile still a WIP

### PR DESCRIPTION
**Pull request by skoshkarli**

### CODE REVIEW

File: src/client/components/MyCloset/Item.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```
Comment (hcnimi) [inline src/client/components/MyCloset/Item.jsx]: being picky, but should we be consistent w/ using ';'?

File: src/client/components/MyCloset/MyClosetItemsContainer.jsx [hunk 2]
```diff
@@ -1,26 +1,116 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {connect} from "react-redux";
```
Comment (hcnimi) [inline src/client/components/MyCloset/MyClosetItemsContainer.jsx]: Suggest adding spaces within the curly braces for consistency.

File: src/client/reducers/myClosetReducer.js [hunk 3]
```diff
@@ -73,6 +76,14 @@ export function myClosetReducer(state = initialState, action) {
       return Object.assign({}, state, {
```
Comment (hcnimi) [inline src/client/reducers/myClosetReducer.js]: are you missing a case here for ITEM_SEASONS?

File: src/client/components/MyCloset/MyClosetFilters.jsx [hunk 4]
```diff
@@ -44,7 +46,8 @@ export class MyClosetFilters extends React.Component {
       {key: 'Levi\'s', value: 'Levi\'s', text: 'Levi\'s'},
       {key: 'Salvatore Ferragamo', value: 'Salvatore Ferragamo', text: 'Salvatore Ferragamo'}
     ];
-    const itemSeasons = [{key: 'Summer', value: 'Summer', text: 'Summer'},
+    const itemSeasons = [
```
Comment (hcnimi) [inline src/client/components/MyCloset/MyClosetFilters.jsx]: How about we move the example data to their own files? I'm doing that for outs and could potentially pull from your items data.

File: src/client/components/MyCloset/Item.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import { Card, Image } from 'semantic-ui-react'
+
+const Item = (props) => (
+  <Card centered raised={true}>
+    <Image src='https://uniqlo.scene7.com/is/image/UNIQLO/goods_65_400495?$prod$' />
```
Comment (hcnimi) [inline src/client/components/MyCloset/Item.jsx]: How about we move the example data to their own files? I'm doing that for outs and could potentially pull from your items data.
Did you hard code the jacket on purpose?

File: src/client/components/MyCloset/MyClosetItemsContainer.jsx [hunk 6]
```diff
@@ -1,26 +1,116 @@
...
-  );
+export class MyClosetItemsContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.filterItems=this.filterItems.bind(this);
+  }
+
+  componentDidMount() {
+    //get items from db based on filters
+    const allItems = [
```
Comment (hcnimi) [inline src/client/components/MyCloset/MyClosetItemsContainer.jsx]: How about we move the example data to their own files? I'm doing that for outs and could potentially pull from your items data.

File: src/client/reducers/myClosetReducer.js (hunk 3 above)
Comment (skoshkarli) [reply src/client/reducers/myClosetReducer.js]: no i am not updating seasons. they are hardcoded

File: src/client/components/MyCloset/Item.jsx (hunk 5 above)
Comment (skoshkarli) [reply src/client/components/MyCloset/Item.jsx]: yeah hard coded on purpose for now. but i'll move it to exampledata file

**Pull request merged**